public/js/apps/management/app.jsx

```diff
@@ -118,7 +120,18 @@ export default class ManagementApp extends Component {
           {...boundPayMethodActions}
         />
       ));
-    } else if (connector.management.view === 'SHOW_SUBSCRIPTIONS') {
+    } else if (connector.management.view === 'SHOW_CONFIRM_DEL_PAY_METHOD') {
+      children.push((
+        <ConfirmDelPayMethod
+          payMethods={connector.user.payMethods}
+          payMethodUri={connector.user.tmp.payMethodUri}
```

Comment:
when seeing this in use, it makes less sense as a user property. Maybe something tied more to the view would make sense, like `connector.management.viewData.payMethodUri`...